docs/codecov.md

```diff
@@ -0,0 +1,55 @@
+# New Relic PHP Agent code coverage monitoring
+
+New Relic PHP Agent's CI/CD is integrated with [Codecov](https://about.codecov.io/)
+to provide insights into agent's automated tests coverage. Insights can be accessed
+through via Codecov's dashboard for [`newrelic-php-agent`](https://app.codecov.io/gh/newrelic/newrelic-php-agent).
+
+## Codecov's dashboard for [`newrelic-php-agent`](https://app.codecov.io/gh/newrelic/newrelic-php-agent)
+
+Codecov's dashboard for [`newrelic-php-agent`](https://app.codecov.io/gh/newrelic/newrelic-php-agent) has many
+useful views. Here's a list of most useful ones:
+* [Coverage by PHP version](https://app.codecov.io/gh/newrelic/newrelic-php-agent/flags)
+* [Coverage by pull requests](https://app.codecov.io/gh/newrelic/newrelic-php-agent/pulls)
+* [Coverage by commit by branch](https://app.codecov.io/gh/newrelic/newrelic-php-agent/commits)
+* [Coverage by file by branch](https://app.codecov.io/gh/newrelic/newrelic-php-agent)
+
+## New Relic PHP Agent's CI/CD integration with [Codecov](https://about.codecov.io/)
+
+New Relic PHP Agent's CI/CD integration with [Codecov](https://about.codecov.io/)
+allows to establish code coverage baseline for long lasting branches,
+which are targets for pull requests, as well as pull request comments,
+which allow to quickly analyze PR's coverage without leaving GitHub's UI.
+
+### Establishing code coverage baseline
+
+The baseline code coverage is used to calculate pull requests code coverage deltas.
+Any branch which is a target for pull request, needs to have a baseline code
+coverage established. Otherwise Codecov's pull request comment will include a
+following message:
```

Comment:
```suggestion
The baseline code coverage is used to calculate a pull request's code coverage deltas.
Any branch which is a target for a pull request needs to have a baseline code
coverage established. Otherwise, Codecov's pull request comment will include the
following message:
```